tests/framework/db/cubrid/QueryBuilderTest.php

```diff
@@ -107,6 +107,10 @@ public function upsertProvider()
         foreach ($concreteData as $testName => $data) {
             $newData[$testName] = array_replace($newData[$testName], $data);
         }
+
+        // skip test
+        unset($newData['no columns to update']);
```

Comment:
Why skipping it? It was changed in the CUBRID builder.

Reply:
Yes, but I don't have cubrid to run tests with it.

Reply:
OK but it should be removed before merge.

Reply:
In this case, we need to add the expected sql query, but I don't know what the correct query is and whether it matches the actual query without running the test.

Reply:
As for CUBRID, that is acceptable :)